dashboard/lib/user_multi_auth_helper.rb

```diff
@@ -1,4 +1,25 @@
 module UserMultiAuthHelper
+  def oauth_tokens_for_provider(provider)
+    if migrated?
+      authentication_option = AuthenticationOption.find_by(
+        credential_type: provider,
+        user_id: id
+      )
```

Comment:
This module is included on `User` so instead you can use the User association where the user id is implied:
```rb
authentication_option = authentication_options.find_by(credential_type: provider)
```